doc/gov/chapters/chapter09.md

```diff
@@ -1,135 +1,157 @@
+*	Purpose of field trials
+*	Goals/desired outcomes of the field trials
+*	Success indicators 
+*	Intentions and expectations
+*	Action plan
+*	Resource requirements
+*	Metrics definition
 
-<a name="9.2.1"></a>
-### 9.2.1 Conformance Levels
+<a name="9.2"></a>
+## 9.2 Adoption Strategy
 
-- **Fully Conformant**: VNFs/CNFs or NFVI are written and designed to be fully conformant to CNTT specification with no use of any of the allowed Exceptions.
-- **Conformant with Exceptions**: VNFs/CNFs or NFVI are written and designed to be conformant to CNTT with one or more of the allowed Exceptions used.
+<a name="9.3.1"></a>
+### 9.2.1 Expectations from Operators
 
 <a name="9.2.2"></a>
-### 9.2.2 Exception Types
-
-- **Technology Exceptions** : Using specific technologies that are considered non conformant to CNTT principles (such as PCIe Direct Assignment, exposure of hardware features to VNFs/CNFs).
-- **Version Exceptions**: Using Versions of  Software components, , APIs, or Hardware that are different from the one specified in the specification.
+### 9.2.2 Expectations from Vendors
 
 <a name="9.2.3"></a>
-### 9.2.3 Transition Framework
+### 9.3.3 Expectations from Industry
 
-#### VNF/CNF Transition Plan Framework 
+<a name="9.3"></a>
+## 9.3 Adoption Roadmap
 
-Exceptions will be clearly recorded in the appropriate specification Appendix which will act as a guidance to VNFs/CNFs vendors of what Exceptions will be allowed in each CNTT release. **Figure 1** below demonstrate the concept.
+<a name="9.4"></a>
+## 9.4 Transition Plan
 
-- As technology matures, fewer and fewer Exceptions will be allowed in CNTT releases.
-- For each CNTT Release, VNF/CNF can be either:
-  - **Fully Conformant**: No Exception used.
-  - **Conformant with Exception**: One or More of the allowed Exceptions in RM has been used.  
+A Transition plan is needed to address technology components that do not presently conform to CNTT mandates, and hence require explicit direction on how the situation will be treated in the present, as well as plans for the future.  The plans might be that the component will be added to the CNTT corpus in a future release, or remain outside of the main body, depending on the nature of the given technology or system.  For example, a technology might be propriatary to a specific vendor, yet has become a de facto standard  would not be part of the reference, but might be referred to due to its wildspread adoption by the industry.
 
+The transition plan described here informs application designers on how the Reference Compliance, and ultimately the industry certification programs will manage and document exceptions encountered during the qualification process.  The actions taken might include flagging warnings and potential errors caused by the variance from the CNTT conformance levels, which could prevent issuance of a certification badge.
```

Comment:
what is "qualification process"?

Reply:
I tried to clarify a bit @pgoyal01 hope it is better now.